_includes/docs/reference/gateway-mqtt-api.md

```diff
@@ -173,6 +173,75 @@ Once command is processed by device, gateway can send commands back using follow
 
 where **$request_id** is your integer request identifier, **Device A** is your device name and **method** is your RPC method name. 
 
+### Get session limits RPC
+
+In order to send RPC commands to server, send PUBLISH message to the following topic:
```

Comment:
This document needs to be corrected. We don't need to explain how to make the client-side RPC again. We need a block with "Reserved RPC calls" and mention the request and response examples. Also, add info about what each of the items in the response JSON actually means.